### PR TITLE
Fix duplicate network on Infrastructure Mapping

### PR DESCRIPTION
**Pull request by priley86**
Deduplicate infra mappings networks on the Infra Mapping List and update counts correctly. Change the `Completed` label to`Created`.

Also incorporates #403 

Also fixes i18n issue... cc: @himdel 

screen shot 2018-06-11 at 10 45 05 am

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ module.exports = {
   globals: {
     $: true,
     __: true,
+    n__: true,
```
Comment (AparnaKarve) [inline .eslintrc.js]: @priley86 Thanks, this looks good.
I have one concern though -- this won't work with Gaprindashvili
@himdel Can you confirm?

I was thinking that we could add the `n` implementation after G is released, just to keep things easier right now with our `gaprindashvili` branch.

Once G is released, we could apply the `n` changes all over the app.

Comment (priley86) [reply .eslintrc.js]: @himdel @AparnaKarve can you please discuss it and give me the suggested solution for now? I would like to get past this one :)

Comment (priley86) [reply .eslintrc.js]: we can revisit this in a future PR. Since `gaprindashvili` does not currently support, I am reverting this change.

Comment (AparnaKarve) [reply .eslintrc.js]: @priley86 Thanks!

Comment (himdel) [reply .eslintrc.js]: `n__` has been supported since 2015 in our JS.

Why would it not work in gaprindasvhili?

**Comment by himdel** [comment]
Not seeing any i18n changes, did this move to a different PR?

File: .eslintrc.js (hunk 1 above)
Comment (mzazrivec) [reply .eslintrc.js]: Where does the thing about `n__` not working in gaprindashvili come from? We have had it for quite a while actually.

Comment (priley86) [reply .eslintrc.js]: @AparnaKarve - i will add the `n__` syntax back in a separate PR later today and let you confirm it once more...

Comment (AparnaKarve) [reply .eslintrc.js]: `n__` has never been used before in our ui-classic JS sources yet, hence wanted to be doubly sure that it was OK for gaprindashvili.

@priley86  apologies for these extra i18n iterations.

With the release impending for Gaprindashvili, just being extra cautious that a wrong commit does not get in 

Thanks everyone!

**Comment by bthurber** [comment]
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1572169

**Pull request merged**